.github/workflows/go-integration.yml

```diff
@@ -0,0 +1,89 @@
+name: Go End-to-End
+
+on:
+  push:
+    branches-ignore:
```

Comment:
These on: values are good for testing; make them look like the ones in go.yaml for merge